### DISCUSSION TITLE
Installer/updater script PR?

### DISCUSSION THREAD

**Discussion by Denperidge** [yokoffing/Betterfox#313] [Ideas]
Hi! Wanted to check - if a sufficiently functional installer/updater script can be created - if that is something that could be Pull Requested? Or is this something that either way you’d want to keep separate from this repo?

**Comment by yokoffing** [comment]
I'm open to creating a space for an installer or updater. I don't have the time or ability to craft one at the moment. But I'm open to the community providing one and allowing others to test it out.

related: https://github.com/yokoffing/Betterfox/issues/167

**Reply by Denperidge** [reply]
If I may be as forward as to ask for your input: I already did a small draft of how it could look in Python (and as such run on most unix based systems as well as - with a pytoexe thing - windows), but I could also see to fork the updater.sh/updater.bat from [arkenfox's user.js repository](https://github.com/arkenfox/user.js), sending any improvements upstream too of course

**Reply by yokoffing** [reply]
Whatever is easiest for the greatest number of people. If you link it, I can put it in the next release notes for the community to test out.

**Reply by Denperidge** [reply]
hi! I just opened a Pull Request before realising that you might have preferred a link to test in the notes first. In that case, see [install.py here](https://github.com/Denperidge-Contrib/Betterfox/blob/installer-cli-squashed/install.py)
